prometheus-alertmanager.yaml

```diff
@@ -45,6 +44,35 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    dependencies:
```

Comment:
Should we add runtime deps? https://eng.inky.wtf/docs/container-images/playbooks/bitnami/#runtime-dependencies

Reply:
given the entrypoint is `/opt/bitnami/alertmanager/bin/alertmanager` we don't need any runtime dependencies. If it was scripts, then we needed some but in this case we can omit this.

Reply:
I didn't notice that, thanks!